app/services/data/get-individual-overview.js

```diff
@@ -14,7 +14,7 @@ module.exports = function (id) {
            'workload_points_calculations.available_points AS availablePoints',
            'workload_points_calculations.total_points AS totalPoints',
            'workload.total_cases AS cases',
-           'workload_points_calculations.contracted_hours AS contractedHours',
+           'workload_owner.contracted_hours AS contractedHours',
```

Comment:
Going to create and extra method in the probation rules that will get the default contracted hours if they are null.